python/ray/autoscaler/commands.py

```diff
@@ -324,6 +323,12 @@ def exec_cluster(config_file, cmd, screen, tmux, stop, start,
         [],
         "",
     )
+    if docker:
+        cname = config["docker"]["container_name"]
+        if not cname:
```

Comment:
Should the docker command just default to the regular exec if no docker config is present? Just a thought.

Reply:
oh, and just raise a warning? would this cause more undetected user errors?

Reply:
Yeah that's what I thought. Maybe it's better to keep it explicit to avoid errors/confusions though.

Reply:
Sounds good.